package.json

```diff
@@ -37,14 +37,9 @@
   "devDependencies": {
     "@gravitational/build": "workspace:*",
     "@ianvs/prettier-plugin-sort-imports": "^4.4.1",
-    "@storybook/addon-actions": "^8.6.12",
-    "@storybook/addon-controls": "^8.6.12",
-    "@storybook/addon-toolbars": "^8.6.12",
-    "@storybook/components": "^8.6.12",
-    "@storybook/preview-api": "^8.6.12",
```

Comment:
these are part of storybook core now: https://storybook.js.org/docs/migration-guide